src/System.Text.Json/src/System/Text/Json/Node/JsonNode.TraversalHelpers.cs

```diff
@@ -17,7 +17,7 @@ public abstract partial class JsonNode
             KeyValuePair<string, JsonNode> nodePair,
             ref Stack<KeyValuePair<string, JsonNode>> currentNodes,
             ref JsonNode toReturn,
-            DuplicatePropertyNameHandling duplicatePropertyNameHandling = DuplicatePropertyNameHandling.Replace)
+            DuplicatePropertyNameHandlingStrategy duplicatePropertyNameHandling = DuplicatePropertyNameHandlingStrategy.Replace)
```

Comment:
Why don't all callers of `AddToParent` pass in a `DuplicatePropertyNameHandlingStrategy`? To put another way, does it make sense for this parameter to be optional?